cf/read_write/um/umread.py

```diff
@@ -637,8 +642,10 @@ def __init__(
 
         int_hdr = rec0.int_hdr
         self.int_hdr_dtype = int_hdr.dtype
-
+        self.real_hdr_dtype = rec0.real_hdr.dtype
         int_hdr = int_hdr.tolist()
+
+        self.real_hdr_dtype = rec0.real_hdr.dtype
```

Comment:
There is a duplicate call above, and the other line looks like the most sensible location for it (after the `int` equivalent variable).

```suggestion
```